common/djangoapps/enrollment/views.py

```diff
@@ -311,48 +337,88 @@ class EnrollmentListView(APIView, ApiKeyPermissionMixIn):
                 * name: Name of the attribute
                 * value: Value of the attribute
 
+            * is_active: Optional. A Boolean value that indicates whether the
+              enrollment is active. Only server-to-server requests can
+              deactivate an enrollment.
+
+            * mode: Optional. The course mode for the enrollment. Individual
+              users cannot upgrade their enrollment mode from "honor". Only
+              server-to-server requests can enroll with other modes.
+            
+            * user: Optional. The user ID of the currently logged in user. You
+              cannot use the command to enroll a different user.
+
         **Response Values**
 
-            A collection of course enrollments for the user, or for the newly created enrollment.
-            Each course enrollment contains:
+            If honor mode is not supported for the course, an HTTP x "Message"
```

Comment:
I'm not familiar with this code, so I'm having a hard time seeing any special logic for honor modes. It looks to me like all invalid requests return a 400 error with the message:

    An error occurred while retrieving enrollments for user '{username}'

I recommend checking with @clintonb or @wedaly.